src/System.Management.Automation/help/ProviderContext.cs

```diff
@@ -110,14 +110,38 @@ internal MamlCommandHelpInfo GetProviderSpecificHelpInfo(string helpItemName)
             // Does the provider know how to generate MAML.
             CmdletProvider cmdletProvider = providerInfo.CreateInstance();
             ICmdletProviderSupportsHelp provider = cmdletProvider as ICmdletProviderSupportsHelp;
+
+            // Under JEA sessions the resolvedProviderPath will be null, we should allow get-help to continue.
             if (provider == null)
             {
                 return null;
             }
 
+            bool isJEASession = false;
+            if (this._executionContext.InitialSessionState.Providers != null && providerInfo != null)
```

Comment:
I just checked and it turns out that ExecutionContext.InitialSessionState property can be null.  Please add a check for this.